### DISCUSSION TITLE
Custom Domain.

### DISCUSSION THREAD

**Discussion by hansputera** [vercel/vercel#5530] [Help] [answered]
Hello There, I just used a custom domain on vercel dashboard. However I use it in CloudFlare and follow the steps provided by the instructions on the dashboard. I've added CNAME with route **api** with target **cname.vercel-dns.com** but when I refreshed my dashboard vercel, vercel said **Invalid Configuration** even though I had correctly followed the steps. Please help :pray:

**Comment by mcsdevv** [comment, marked as answer]
Hello, please see the following article: https://vercel.com/knowledge/using-cloudflare-with-vercel

**Reply by hansputera** [reply]
still.

**Reply by Aichnerc** [reply]
Same. Set CNS record, however not nameservers - for obvious reasons.

**Reply by hansputera** [reply]
right, fixed.
